src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,28 @@
+import { Todo } from '../../types/Todo';
+import { GlobalLoader } from '../../types/GlobalLoader';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  globalLoader: GlobalLoader,
```

Comment:
Naming is not perfect - global loader sounds like a full screen something
```suggestion
  todoLoader: GlobalLoader,
  // or
  isTempTodoLoading: boolean
  // etc
```